apisix/cli/ngx_tpl.lua

```diff
@@ -154,6 +154,10 @@ http {
     }
     {% end %}
 
+    {% if enabled_plugins["error-log-logger"] then %}
+        lua_capture_error_log  1m;
```

Comment:
`1m` is small, please change to a larger size.

Reply:
change to 10m